bin/spark-evaluate.js

```diff
@@ -51,11 +90,18 @@ await Promise.all([
     fetchRoundDetails,
     recordTelemetry,
     createPgClient,
-    logger: console
+    logger: console,
+    cancelStuckTransactions
   }),
-  startCancelStuckTxs({
-    walletDelegatedAddress,
-    address: wallet.address,
-    signer
-  })
+  (async () => {
+    while (true) {
+      try {
+        await cancelStuckTransactions.olderThan(2 * ROUND_LENGTH_MS)
+      } catch (err) {
+        console.error(err)
+        Sentry.captureException(err)
+      }
+      await timers.setTimeout(CHECK_STUCK_TXS_DELAY)
+    }
+  })()
```

Comment:
Let's move this to a named function please, that function should be exported by the same file that exports the main implementation (see one of my comments above).